Resources/ServerInfo/Guidebook/Science/Cyborgs.xml

```diff
@@ -0,0 +1,66 @@
+<Document>
+  # Cyborgs
+  Cyborgs are man-machine hybrids, purpose-built by the [textlink="robotics"  link="Robotics"] division of science for various duties across the station. Bound by [color=cyan]silicon laws[/color] which they follow strictly, and religiously, they serve the crew unconditionally. [italic](Most of the time.)[italic]
+
+  ## Basic Components
+  <Box>
+    <GuideEntityEmbed Entity="CyborgEndoskeleton" Caption="Cyborg Endoskeleton"/>
+    <GuideEntityEmbed Entity="ExosuitFabricator" Caption="Exosuit Fabricator"/>
+  </Box>
+  All cyborgs are built on a endoskeleton, which can be crafted at an [color=#a4885c]Exosuit Fabricator[/color]. Further assembly steps can be followed by [color=#a4885c]examining[/color] it. Once built, further upgrades, such as additional tools, and longer battery life can be done with modules and a better power cell, respectively.
+
+  In addition, newly built cyborgs need a [color=#a4885c]brain[/color] able to interface with the electronic body. There are two main types of such, both of which are able to speak independently, should the chassis be decommissioned, exchanged, or otherwise [color=red]suffer an unscheduled dissassembly[/color];
+  - The [color=#a4885c]Man-Machine Interface[/color] or MMI for short, is an apparatus able to faciliate communication between a biological brain and electronic components, enabling those suffering from major workplace accidents to continue serving their crew, long after their body is gone.
+  - The [color=#a4885c]Positronic Brain[/color] or PB for short, is an entirely artificial brain, capable of spotaneous neural activity. The time for the synthetic neurons to descramble can wildly vary, but they allow for the creation of cyborgs without having to extract a brain from a crewmember.
+  <Box>
+    <GuideEntityEmbed Entity="MMIFilled" Caption="Man-Machine Interface"/>
+    <GuideEntityEmbed Entity="PositronicBrain" Caption="Positronic Brain"/>
+  </Box>
+  Both brains can be fabricated without requiring any additional research.
+
+  ## Chassis
+  While all cyborgs share the same endoskeleton, not all share the same [color=#a4885c]chassis[/color]. The chassis determines what [color=#a4885c]modules[/color] the cyborg can have, along with the [color=#a4885c]departmental radio channel[/color] they correspond to. By default, they will always have access to [color=violet]Science[/color] and [color=green]station-wide[/color] frequencies, along with having [color=#a4885c]all-access[/color].
```

Comment:
```suggestion
  While all cyborgs share the same endoskeleton, not all share the same chassis. The chassis determines what modules the cyborg can have, along with the [color=#a4885c]departmental radio channel[/color] they correspond to. By default, they will always have access to [color=#D381C9]Science[/color] and [color=green]station-wide[/color] frequencies, along with having [color=#a4885c]all-access[/color].
```
omitted some of the highlights because it looked kinda excessive.
departmental.yml contains all of the hex colors corresponding to each department. Invaluable resource!